cmd/hyperconverged-cluster-operator/main.go

```diff
@@ -168,6 +172,14 @@ func main() {
 	err = createPriorityClass(ctx, mgr)
 	cmdHelper.ExitOnError(err, "Failed creating PriorityClass")
 
+	// Setup Monitoring
+	operatormetrics.Register = controllerruntimemetrics.Registry.Register
+	err = metrics.SetupMetrics()
+	if err != nil {
+		logger.Error(err, "failed to setup metrics: %v")
```

Comment:
@machadovilaca @dharmit and not sure if I should use `cmdHelper.ExitOnError` or `logger.Error` WDYT?

Reply:
I think `cmdHelper.ExitOnError` because `logger.Error` is so far being used in this file when we want to do something extra besides logging and exit, e.g., emit an event or set upgradeable condition, whereas `cmdHelper.ExitOnError` has been used each time we want to log the error and exit.

In this case, since we're only logging and then exiting, `cmdHelper` seems apt to me.

My comment is based on the pattern I observed in this file by doing Ctrl+F.

Reply:
I agree, updated to cmdHelper.ExitOnError.